charts/sonarqube/README.md

```diff
@@ -287,8 +287,8 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `jvmOpts` | Values to add to SONARQUBE_WEB_JVM_OPTS | `""` |
 | `jvmCeOpts` | Values to add to SONAR_CE_JAVAOPTS | `""` |
 | `sonarqubeFolder` | Directory name of Sonarqube | `/opt/sonarqube` |
-| `sonarProperties` | Custom `sonar.properties` file | `None` |
-| `sonarSecretProperties` | Additional `sonar.properties` file to load from a secret | `None` |
+| `sonarProperties` | Custom `sonar.properties` values (e.g., "sonarProperties.sonar.forceAuthentication=true") | `None` |
```

Comment:
I would use "key-value pairs" here instead of values to align on the two properties

Reply:
yes it makes sense